src/main/java/edu/harvard/iq/dataverse/DatasetVersionServiceBean.java

```diff
@@ -918,10 +911,7 @@ public boolean doesChecksumExistInDatasetVersion(DatasetVersion datasetVersion,
         Query nativeQuery = em.createNativeQuery(query);
         List<String> checksumList = nativeQuery.getResultList();
```

Comment:
Native queries are inherently type unstable, so it wouldn't help, but as a stole thing, sure